frontend/src/app/testQueue/AoEForm/AoEModalForm.jsx

```diff
@@ -19,7 +19,13 @@ const AoEModalForm = ({
 }) => {
   const [modalView, setModalView] = useState(null);
   const [patientLink, setPatientLink] = useState(qrCodeValue);
+  const [smsSuccess, setSmsSuccess] = useState(false);
   const modalViewValues = [
+    {
+      label: "Text message",
+      value: "text",
+      checkedLabel: patient.telephone,
```

Comment:
Instead of implementing `checkedLabel` can we make the label a `ReactNode` and pass in the extra markup that way? That would make line 25:
```
label: <>
  Text message 
  <span className="usa-checkbox__label-description">
     {patient.telephone}
  </span>
</>,
```
Assuming I didn't screw up something

Reply:
We could. What's the advantage of doing it this way? FYI this project is my first time using React, so I'll have some basic questions.

Reply:
The main advantage is that the generalized component doesn't continue to grow to handle special cases in label presentation. For example, if the design becomes "Text message **(919) 555-1212** _(continental USA only)_" we don't have to figure out how to sneak more styling into here. There's no reason why a `<label />` can't have markup, we just started with it declared as `string` because we didn't need styling before this. 

In a more general sense this is an _Inversion of control_ pattern. Instead of telling the component what to do in specialized situations, we can have the component let us do it.

Reply:
Makes sense to me, thanks. Doing it this way, would we revert the changes to `RadioGroup.tsx`?

Reply:
Updated in https://github.com/CDCgov/prime-simplereport/pull/869/commits/25989e3ec6fcbe049a449f4c4ab6f44fd7d2e5d8

Reply:
We'd need to change the type to `label: React.ReactNode` but otherwise shouldn't need other changes.

Reply:
Updated in https://github.com/CDCgov/prime-simplereport/pull/869/commits/fc45d10ab117a3e321b45b409427ad8833d9dc49

Reply:
When I did that I got the error:
```
Line 1:17:  'ReactNode' is defined but never used  @typescript-eslint/no-unused-vars
```

Do we also need to add:
```
type NewType = React.ReactNode;
```

Reply:
nvm should be fixed in https://github.com/CDCgov/prime-simplereport/pull/869/commits/193818a977db1a3f91ea12707ba33f92d88c2fe5